### PR TITLE
Port global reach page to react

### PR DESCRIPTION
**Pull request by RoyEJohnson**
Issue #1786
Based on Impact page PR, so don't merge this until that one is done.

### CODE REVIEW

File: src/app/pages/global-reach/statistics/stat.js [hunk 1]
```diff
@@ -1,13 +1,29 @@
...
+        </div>
+    );
+}
 
-export default componentType(spec);
+export default function Statistics({cards}) {
+    return (
+        <div className="stat-bg">
+            <div className="statbox">
+                {cards.map((card) => <Card item={card} />)}
```
Comment (nathanstitt) [inline src/app/pages/global-reach/statistics/stat.js:25]: 🔑

File: src/app/pages/global-reach/statistics/stat.scss [hunk 2]
```diff
@@ -1,86 +1,90 @@
...
-    }
+        ::-webkit-scrollbar { display: none; }
 
-    .col-img {
-        border-radius: 0.3rem 0.3rem 0 0;
-        width: 100%;
-    }
+        .col1 {
+            border-radius: 0.3rem; // background: linear-gradient(224.64deg, rgba(250,188,77,0) 0%, #FABC4D 100%), linear-gradient(0deg, #F4D019 0%, #F4D019 100%);
+            box-shadow: 0 1.5rem 4rem 0 rgba(0, 0, 0, 0.2);
```
Comment (nathanstitt) [inline src/app/pages/global-reach/statistics/stat.scss:27]: commented rule that could be removed

File: src/app/pages/impact/section/founding.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import {RawHTML} from '~/components/jsx-helpers/jsx-helpers.jsx';
+import './founding.css';
+
+export default function Founding({heading, description, portrait, caption}) {
+    return (
+        <Section id="founding">
+            <div className="text-overlapping-photo">
+                <img src={portrait.image} alt={portrait.altText} />
+                <HeadingAndDescription
+                    classList={['text-block', 'card']}
```
Comment (nathanstitt) [inline src/app/pages/impact/section/founding.js:12]: does the array need a `.join(' ')` call?

File: src/app/pages/impact/section/reach.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        </div>
+    );
+}
+
+export default function Reach({heading, description, facts}) {
+    return (
+        <Section id="reach">
+            <HeadingAndDescription classList={['text-block']} {...{heading, description}} />
+            <div class="fact-boxes scroll-on-mobile">
+                {facts.map((fact) => <FactBox fact={fact} />)}
```
Comment (nathanstitt) [inline src/app/pages/impact/section/reach.js:24]: 🔑

File: src/app/pages/impact/section/sustainability.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    return (
+        <Section id="sustainability" className="white">
+            <HeadingAndDescription {...{heading, description}} />
+            <div class="scroll-on-mobile">
+                <div class="icon-row">
+                    {
+                        partners.map((icon) =>
+                            <img src={icon.image.image}
+                                alt={icon.imageAltText || icon.image.altText}
+                            />
```
Comment (nathanstitt) [inline src/app/pages/impact/section/sustainability.js:14]: 🔑

File: test/src/pages/adoption/how-using.test.js [hunk 6]
```diff
@@ -1,71 +1,29 @@
...
-            () => props,
-            (newValue) => {
-                usingInfo = newValue;
-            }
-        ));
+        wrapper = makeMountRender(HowUsing, props)();
+        setTimeout(() => {
+            wrapper.update();
+            done();
+        }, 10);
```
Comment (nathanstitt) [inline test/src/pages/adoption/how-using.test.js:20]: you can return a promise in `beforeEach` and it'll wait for it to resolve before running specs

File: src/app/pages/impact/section/founding.js (hunk 3 above)
Comment (RoyEJohnson) [reply src/app/pages/impact/section/founding.js:12]: No, the join happens in the component.

File: test/src/pages/adoption/how-using.test.js (hunk 6 above)
Comment (RoyEJohnson) [reply test/src/pages/adoption/how-using.test.js:20]: That's what `done` is for: you call it to resolve a promise that `beforeEach` creates for you. The timeout was to allow an internal-to-the-component async event to happen. But I discovered I could move the `update` call out of `beforeEach` and into the test and get the same effect.

**Pull request merged**